base/docs/helpdb/Base.jl

```diff
@@ -1489,7 +1489,7 @@ This function is typically used to load library code, and is implicitly called b
 load packages.
 
 When searching for files, `require` first looks for package code under `Pkg.dir()`, then tries
-paths in the global array `LOAD_PATH`.
+paths in the global array `LOAD_PATH`. `require` is case-sensitive  on all platforms including those with case-insensitive filesystems like macOS and Windows.
```

Comment:
NTFS **is** case-sensitive! I did not know that. 

From [Wikipedia](https://en.wikipedia.org/wiki/Case_sensitivity) 

> Current Windows file systems, like NTFS, are case-sensitive; that is a readme.txt and a Readme.txt can exist in the same directory. However, the API for file access in Windows applications is case-insensitive, which makes filenames case-insensitive from the application's point of view.

I am not sure whether I should keep windows.